src/sentry/metrics/minimetrics.py

```diff
@@ -102,8 +91,6 @@ def before_emit_metric(key: str, tags: Dict[str, Any]) -> bool:
 class MiniMetricsMetricsBackend(MetricsBackend):
     def __init__(self, prefix: Optional[str] = None):
         super().__init__(prefix=prefix)
-        if not have_minimetrics:
```

Comment:
I think the entire constructor can go now.

Reply:
Good point